fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoredRecord.java

```diff
@@ -150,6 +150,25 @@ public static <M extends Message> FDBStoredRecordBuilder<M> newBuilder(@Nonnull
         return new FDBStoredRecordBuilder<>(protoRecord);
     }
 
+    /**
+     * Get a builder with the parameters of a given {@link FDBStoredRecord}.
+     * @param record given record
+     * @param <M> type used to represent stored records
+     * @return a new builder
+     */
+    @Nonnull
+    public static <M extends Message> FDBStoredRecordBuilder<M> newBuilder(@Nonnull FDBStoredRecord<M> record) {
```

Comment:
I believe that a method (usually not `static`, but that is secondary) from a built instance back to a builder is `asBuilder`.